dev/com.ibm.ws.kernel.service/resources/com/ibm/ws/kernel/service/utils/resources/ServiceMessages.nlsprops

```diff
@@ -56,6 +56,10 @@ invalidDuration=CWWKE0107W: The specified parameter does not represent a duratio
 invalidDuration.explanation=The value for the specified configuration parameter is not, or cannot be converted to, a duration value.
```

Comment:
Is the message correct? It ends with "duratio"

Reply:
@helyarp Thanks for the review, updated all the others, but this one confused me because the comment is on a message that I didn't modify (and also I do see duration). The modified one is: CWWKE0120E